resin/models/device.py

```diff
@@ -530,28 +534,6 @@ def register(self, app_name, uuid):
             endpoint=self.settings.get('pine_endpoint'), login=True
         )
 
-    def restart(self, uuid):
```

Comment:
Yeah, I was wrong, only the function name was changed. I removed the commit for this change. @dfunckt can you check the PR again?